src/app/nodetool/qgsmaptoolnodetool.cpp

```diff
@@ -360,7 +360,16 @@ void QgsMapToolNodeTool::updateSelectFeature()
 
   mSelectRubberBand = new QgsGeometryRubberBand( mCanvas, mSelectedFeature->geometry()->type() );
   mSelectRubberBand->setBrushStyle( Qt::SolidPattern );
-  mSelectRubberBand->setFillColor( QColor( 255, 0, 0, 50 ) );
+
+  QSettings settings;
+  QColor color(
+    settings.value( "/qgis/digitizing/select_color_red", 255 ).toInt(),
```

Comment:
Is there a reason why "select_color_red/green/blue" is used here, while in the options screen, only "line_color_red/green/blue" can be configured?